clocks/users/redis_client.py

```diff
@@ -0,0 +1,22 @@
+from django.core.cache import cache
+
+
+def save_new_client_to_cache(token, nickname: str, role: str, ttl: int = 432000) -> None:
```

Comment:
```suggestion

DEFAULT_TOKEN_TTL = 60 * 60 * 24 * 5  # 5 days

def save_new_client_to_cache(token, nickname: str, role: str, ttl: int = DEFAULT_TOKEN_TTL) -> None:
```

так гораздо понятнее будет